optaplanner-docs/src/main/asciidoc/UseCasesAndExamples/MeetingScheduling/MeetingScheduling-section.adoc

```diff
@@ -13,6 +13,8 @@ Hard constraints:
 
 * Room conflict: two meetings must not use the same room at the same time.
 * Required attendance: A person cannot have two required meetings at the same time.
+* Required room capacity: A meeting shouldn't be at a room that doesn't fit all of the meeting's attendees.
```

Comment:
"shouldn't" -> hard constraints better use verbs like "must" and "must not".
"Should (not)" is too soft.